src/providers/postgres/qgspostgresprovider.h

```diff
@@ -218,7 +219,8 @@ class QgsPostgresProvider final: public QgsVectorDataProvider
      * \param value the value to convert
      * \returns a QVariant of the given type or a null QVariant
      */
-    static QVariant convertValue( QVariant::Type type, QVariant::Type subType, const QString &value, const QString &typeName );
+    QVariant convertValue( QVariant::Type type, QVariant::Type subType, const QString &value, const QString &typeName ) const;
```

Comment:
What is the purpose of this function? I think everywhere is just the static variation used

Reply:
Inside qgspostgresprovider.cpp there are many calls to convertValue. This is a convenience method so we don't need to touch them to add the new connection parameter in every call.